src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -0,0 +1,387 @@
+     *
+     * @param authors
+     */
+    private void fillContributor(String authors) {
+        AuthorList list = AuthorList.parse(authors);
+        for (Author author : list.getAuthors()) {
+            dcSchema.addContributor(author.getFirstLast(false));
+        }
+    }
+
+    /**
+     * Bibtex-Fields used: author, Field: 'dc:creator'
+     *
+     * @param creators
+     */
+    private void fillCreator(String creators) {
+        AuthorList list = AuthorList.parse(creators);
+
+        for (Author author : list.getAuthors()) {
+            dcSchema.addCreator(author.getFirstLast(false));
+        }
+    }
+
+    /**
+     * Bibtex-Fields used: year, month, Field: 'dc:date'
+     */
+    private void fillDate() {
+        bibEntry.getFieldOrAlias(FieldName.DATE)
+                .ifPresent(publicationDate -> dcSchema.addUnqualifiedSequenceValue("date", publicationDate));
+    }
+
+    /**
+     * Bibtex-Fields used: abstract, Field: 'dc:description'
+     *
+     * @param description
+     */
+    private void fillDescription(String description) {
+        dcSchema.setDescription(description);
+    }
+
+    /**
+     * Bibtex-Fields used: doi, Field: 'dc:identifier'
+     *
+     * @param identifier
+     */
+    private void fillIdentifier(String identifier) {
+        dcSchema.setIdentifier(identifier);
+    }
+
+    /**
+     * Bibtex-Fields used: publisher, Field: dc:publisher
+     *
+     * @param publisher
+     */
+    private void fillPublisher(String publisher) {
+        dcSchema.addPublisher(publisher);
+    }
+
+    /**
+     * Bibtex-Fields used: keywords, Field: 'dc:subject'
+     *
+     * @param value
+     */
+    private void fillKeywords(String value) {
+        String[] keywords = value.split(",");
+        for (String keyword : keywords) {
+            dcSchema.addSubject(keyword.trim());
+        }
+    }
+
+    /**
+     * Bibtex-Fields used: title, Field: 'dc:title'
+     *
+     * @param title
+     */
+    private void fillTitle(String title) {
+        dcSchema.setTitle(title);
+    }
+
+    /**
+     * All others (+ bibtex key) get packaged in the relation attribute
+     *
+     * @param key Key of the metadata attribute
+     * @param value Value of the metadata attribute
+     */
+    private void fillCustomField(String key, String value) {
+        dcSchema.addRelation("bibtex/" + key + '/' + value);
+    }
+
+    public DublinCoreSchema fillDublinCoreSchema() {
+
+        // Query privacy filter settings
+        boolean useXmpPrivacyFilter = xmpPreferences.isUseXMPPrivacyFilter();
+        // Fields for which not to write XMP data later on:
+        Set<String> filters = new TreeSet<>(xmpPreferences.getXmpPrivacyFilter());
+
+        for (Entry<String, String> field : bibEntry.getFieldMap().entrySet()) {
+
+            if (useXmpPrivacyFilter && filters.contains(field.getKey())) {
+                continue;
+            }
+
+            if (FieldName.EDITOR.equals(field.getKey())) {
```

Comment:
I would prefer a switch/case here, but that's just my style. The others don't like them that much ;) So you can change it or leave it.

Reply:
I leave it :)